synaptic_reconstruction/tools/distance_measure_widget.py

```diff
@@ -10,6 +10,7 @@
 
 from .base_widget import BaseWidget
 from .. import distance_measurements
+from .util import compute_scale_from_voxel_size
```

Comment:
I don't think we need this here. The scaling is only needed for the segmentation widget.
(In general: avoid unnecessary imports.)